extension/src/LanguageFunctions.ts

```diff
@@ -226,230 +228,152 @@ export async function findMultipleTargets(): Promise<void> {
-                                if (gXlfMaxWith !== langMaxWith) {
-                                    if (!gXlfMaxWith) {
-                                        logOutput('maxwidth removed for Id ', id);
-                                        langCloneElement.removeAttribute('maxwidth');
-                                    } else {
-                                        logOutput('maxwidth updated for Id ', id);
-                                        langCloneElement.setAttribute('maxwidth', gXlfMaxWith);
-                                    }
-                                    numberOfUpdatedMaxWidths++;
-                                }
-                                if (undefined !== gXlfNoteElement) {
-                                    if (undefined === langNoteElement) {
-                                        logOutput('Note missing for Id ', id);
-                                        let insertBeforeNode = <Element>getNoteElement(langCloneElement, xmlns, 'Xliff Generator');
-                                        langCloneElement.insertBefore(gXlfNoteElement.cloneNode(true), insertBeforeNode);
-                                        langCloneElement.insertBefore(langTempDom.createTextNode(getTextNodeValue(10)), insertBeforeNode);
-                                        numberOfUpdatedNotes++;
-                                    } else {
-                                        if (gXlfNoteElement.textContent !== langNoteElement.textContent) {
-                                            logOutput('Note comment updated for Id ', id);
-                                            langNoteElement.textContent = gXlfNoteElement.textContent;
-                                            numberOfUpdatedNotes++;
-                                        }
-                                    }
-                                }
-                            }
-                            tmpGroupNode.appendChild(langCloneElement);
+                            langTransUnit.target[0].translationToken = TranslationToken.Review;
                         }
                     }
+                    langTransUnit.source = gTransUnit.source;
+                    numberOfUpdatedSources++;
+                }
+                if (langTransUnit.maxwidth !== gTransUnit.maxwidth) {
+                    langTransUnit.maxwidth = gTransUnit.maxwidth;
+                    numberOfUpdatedMaxWidths++;
+                }
+                if (langTransUnit.note[0].textContent !== gTransUnit.note[0].textContent) {
+                    langTransUnit.note[0].textContent = gTransUnit.note[0].textContent;
+                    numberOfUpdatedNotes++;
+                }
+                newLangXliff.transunit.push(langTransUnit);
+                langXliff.transunit.splice(langXliff.transunit.indexOf(langTransUnit), 1);
+            } else {
+                // Does not exist in target
+                if (!sortOnly) {
```

Comment:
Do we really need the sortOnly option in this function with this change? It should be a stand alone operation in my opinion. We have a sort function in the Xliff class. It's not tested i a larger extent tho.

Reply:
Have you figured out how MS sorts the g.xlf file? I thought that was a well hidden mystery! If your sort function sorts exactly as MS sorts g.xlf, then we'll change - otherwise this will stay as is for now :)